lib/active_model/serializer/associations.rb

```diff
@@ -93,7 +93,7 @@ def associations(include_tree = DEFAULT_INCLUDE_TREE)
         Enumerator.new do |y|
           self.class._reflections.each do |reflection|
             next unless include_tree.key?(reflection.name)
-            y.yield reflection.build_association(self, instance_options)
+            y.yield reflection.build_association(self, instance_options.merge(_parent: self.class))
```

Comment:
Good catch, that was a rebase mistake. It is actually unnecessary to supply it here. However, the name is `_parent` because it is passed down to a child serializer later on. The underscore is there because we don't want users to toy with this option.